src/components/sectionsWheelPicker/index.tsx

```diff
@@ -56,7 +54,7 @@ const SectionsWheelPicker = (props: SectionsWheelPickerProps) => {
   const renderSections = () =>
     _.map(sections, (section, index) => {
       return (
-        <View height={1} width={section.style?.width || defaultWidth} key={index} testID={testID}>
+        <View height={1} width={section.style?.width} key={index} testID={testID}>
           <WheelPicker testID={`${testID}.${index}`} {...wheelPickerProps} {...section} style={{width: '100%'}}/>
```

Comment:
Same here, no reason to pass the width, the content of the wheel picker should determine its size. 
Also, regardless, passing the style like that overrides the user style